addon/components/tf-form/component.js

```diff
@@ -0,0 +1,30 @@
+/**
+  @module ember-ticketfly-fields
+ */
+import Ember from 'ember';
+import layout from './template';
+const { Component, guidFor, computed } = Ember;
+
+/**
+  @public
+  @class TfForm
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+  
+  tagName: 'form',
+  classNames: ['tf-form'],
+
+  inputGuid: computed({
+    get() {
+      return `id-${guidFor(this)}`;
+    }
+  }),
+
+  describedByGuid: computed({
+    get() {
+      return `described-by-${guidFor(this)}`;
+    }
+  })
+});
```

Comment:
Should forms need to know about they're inner inputs. I'd imagine it would just yield a block scope for inputs, and then take care of handling its submit action -- perhaps by calling an `onSubmit` closure action.

Here's something I mocked up real quick:

```js
import Ember from 'ember';
import Component from 'ember-component';

const { K: noop } = Ember;

export default Component.extend({
  tagName: 'form',
  classNames: ['u-m0', 'u-overflow-hide'],
  attributeBindings: ['autocomplete', 'novalidate', 'method'],

  autocomplete: 'on',
  novalidate: 'novalidate',
  method: 'post',

  onSubmit: noop,

  submit(ev) {
    ev.preventDefault();

    this.get('onSubmit')(...arguments);
  }

});
```